### DISCUSSION TITLE
Generate PR even if the previous one was closed

### DISCUSSION THREAD

**Discussion by stavrogin** [renovatebot/renovate#8840] [Request Help]
Hi,

I am using the self-hosted version and would like to use a GHE repository to do some tests.

Renovate is working great but for testing purpose I would like to be able to create a PR even if exists a the previous one not merged into master branch. 

This is because I want to do some tests in order to customize the PR message, labels and so on and see the resulting PR on GitHub. It would be not convenient for this purpose to add to the pom.xml (or package.json) an old dependency of some old library every time to trigger a new PR.

Is there any configuration to force the creation of a new PR even if it has been previously skipped?

Thank you

**Comment by rarkins** [comment]
`"recreateClosed": true`

**Reply by stavrogin** [reply]
Thank you!
